backend/utils.py

```diff
@@ -243,12 +175,14 @@ def set_nested_in_dict(d: Dict, key_path: List, value: Any):
 
 def return_err_with_trace(func):
     """Handle exceptions"""
+    # @joeflack4: is this helping us? it's used for four routes. it took me a while to track down
+    #   that it needed to be async and have `await func(..)`
```

Comment:
never mind. it is useful:
![image](https://github.com/jhu-bids/TermHub/assets/1586931/6294b9e3-2f11-41c5-a892-35bc890c2b76)

But we need to fix the front end to take advantage of it.